project.clj

```diff
@@ -0,0 +1,141 @@
+(require '[clojure.string :as string])
```

Comment:
This is a generic project.clj which I use to work with deps.edn based files, allowing me to push releases.

It has working very nicely for all of 2021 it's scary but no real need to review it - it's boilerplate I copy / refine from time to time. It just works :-)